### ISSUE TITLE
Daily status - April 2021

### ISSUE THREAD

**Issue by seanpm2001** [seanpm2001/seanpm2001#7]
***

### Daily status - April 2021

This is the daily status thread for April 2021. Daily status posts will be put daily (or when I get to it) below this post.

[Click here to see last months daily status posts](https://github.com/seanpm2001/seanpm2001/issues/6)

***

**Comment by seanpm2001** [comment]
***

### Status update: April 1st 2021

I had an OK day today. I started with a clean slate of progress as it is a new month. Today, I released a non-traditional April Fools joke project, non-traditional as in it doesn't fool anyone (and if it does, I don't even) I created a project with joke branches, and branch puns. It is also to be used for GitHub branch experimentation.

I quickly archived 2 new monthly releases of my Khan Academy project in less than an hour. Today, I spent the rest of my morning, and a couple hours in the afternoon preparing my new laptop.

I made a "small" update, and added support for 80 new languages. It takes too long to translate them all at once, so I will try to translate 9-10 per day for the next 7-8 days.

I had a lot of procrastination issues today, and didn't get as much done as yesterday. However, I have been making my workload work in my favor as to geasier on myself tomorrow.

My new laptop is fully prepared except for Firefox profiles, video, and music.

I stayed up really late tonight and wasn't too functional. I hope for a better day tomorrow.

Today was a good day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 2nd 2021

I forgot to upload my GitHub status file last night. A double upload was done tonight.

I had an OK day with development. I dealt with addiction problems, but still got a little bit of work done. I got some work done on forked projects, and did my daily documentation work, along with 12 new language translations to my GitHub profile README.md file.

I got my new laptop further set up today. I didn't think copying over Firefox profiles would be so easy, it was pretty much seamless. I successfully cloned my Firefox setup today. It is fully functional too. I was skeptical on what it was going to grab, but it got everything (that I can tell so far, except for 1 thing, I noted that it didn't get the `TITLEBAR=ENABLED` preference. I like it better without the titlebar, I didn't know you could just turn it off on Linux. It blends into the theme.

With my new backup laptop today, I have found out why I haven't been able to get some of the apps I wanted: apparently the Ubuntu store is different from the snap store. The reason why my applications weren't coming up was because I was searching from the snap store, because the Ubuntu store originally redirected me there. I have now customized it further and have stopped using the snap store. One unique program I overlooked was the help center application. I guess the icon was just too generic and I overlooked it. It really gives off Windows XP help center vibes to me (which I am very fond of) and it is an extremely helpful documentation center, not only for beginner and non-tech savvy users, but also for tech savvy and intermediate users.

I have been in the process of getting my projects ready for automation, starting yesterday. I have finally come to terms that the way I am doing things is inefficient, and I have found that automation will be the way to go for some projects. I will have to lay down some ground rules for myself (and possibly others) regarding this, as I shouldn't be 100% reliant on automation (or else bad things will start happening, such as programs becoming Google Translate translating through 1-1000 different languages level of quality) I will try to start laying out rules for that tomorrow.

Today was a OK day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 3rd 2021

I forgot to upload my GitHub status file last night. A double upload was done tonight.

I had an OK day with development. I dealt with procrastination problems, and did my daily documentation work, and added 10 new language translations to my GitHub profile README.md file.

My new laptop is already experiencing new bad problems. I hate complaining about it, but it is important for documentation. Today, I noted that CPU usage was excessive, and I killed tracker-miner-fs, which did nothing to help. I noted that GNOME System Monitor was not reporting correct Daemon/process CPU usage. Most of my CPU cores were using over 22% CPU, yet the process list only showed a single process using 1.0-1.99% CPU, and everything else using <0.99% of the total CPU. I also noticed that Cache usage went up to a maximum of 6.3 GiB (6.76 GB) of memory, but it went down to 4.4 GiB (4.72 GB) when opening Firefox. Something is not right, although I learned more about tracker and how it works today when experimenting with the program. I killed it twice, and found that it now wants to reindex my entire laptop, but won't show more than 0 files being indexed. This issue currently is unresolved.

I had severe procrastination problems today, but got some work done on some projects, including my project automator project.

Today was a OK day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 4th 2021

My gaming sessions were excessive today, and I didn't expect them to be. I was hoping yesterday was the last difficult day of week. I had minor procrastination issues, but I did less work, as I need to go to bed earlier.

I had a really hard time sleeping last night. I didn't get to sleep until past 3:00 am, and I slept in past 12:00 pm (waking up at 12:03 pm) something that is extremely rare for me (2nd time in my life where I slept in past noon)

I got really tired during development and had to wrap up and go to bed. I have a meeting tomrrow morning. I was only abe to do 5 translations today, which is still over 3%

I forked and patially finalized some joke projects today, incluidng Branch Ranch and 404Day.

Today was a OK day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 5th 2021

I had an appointment with my therapist this morning. I won't go into the details.

I decided to do git-image work directly aferwards, along with other projects, before starting my gaming session. I made lots of progress before my gaming session, and a good amount of progress afterwards.

I have been bummed about this for about a week now but haven't mentioned it yet, but I do not like the flatter `README` index button on GitHub. Luckily, I have a copy of the original graphics in both light and dark modes.

Tomorrow may be a difficult day, due to the addicting game that is so addicting that I only allow myself to play it once a month (Idle Miner Tycoon)

I worked pretty heavily on updating the documentation for SNU Beliefs today. I was able to get some important resources and ideas to continue the project.

Today, I also torrented 1 file from archive.org, as the main site was going too slow, and I wanted to save bandwidth.

I did Organization work over Wiki work, I will try again either tomorrow or the day after to create project Wikis for 3 important projects.
 
My goal of going to bed earlier was set back today due to how late I stayed up. The fish tank video process hasn't been as enthusiastic lately, but I have gotten a lot done today other than that.

When creating organizations, I kept getting 404 errors thrown at me, and the forks were disapearing and re-appearing with each refresh. The issue eventually settled, and I don't plan on creating any new organizations for a while.
 
Today was a good day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 6th 2021

I had a good day with development today, despite the monthly addiction. I also had the time to make 10 new translations of my GitHub profile.

Today was the first Tuesday of the month, so I had my planned long Idle Miner Tycoon session. It felt like I was going to be done by 3:00 pm, but I got sucked in further and played until 4:59 pm. It affected GitHub progress today, but I still remained as productive as I could be via dualtasking. This is why I took the risk and got more done yesterday, so that there would be less to focus on today.

I had time to create more commit description batch files today, something I am still behind on. It was required due to the massive amount of batch uploads today.

I got the usual Git-image work done, then went to complete my journaling. I am still behind on updating my marine biology research project (update: I got to this tonight)

SNU has been getting extra development attention lately, I am hoping this will result in a return to normal soon.

Today was an OK day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 7th 2021

I had a good day with development today. Last night, I got feedback on my profile README project about how to continue it with `.gitattributes` which I spent some time researching, testing, and observing today. Along with this, I finished my V9 GitHub profile README.md file translation project, translating the last 22 languages today, with 110 languages now translated. The task is now up-to-date, with no links leading to a 404 page. I have had a lot of time today.

I Worked late again tonight, I am still failing on getting my schedule under control. I had a lot of time to work today, and had brief issues with my `aAa` key earlier. Luckily it went away after less than an hour.

I worked on various projects today, such as creating Project Wikis for 5 projects, sorting projects and tabs, writing documentation, and creating a virtual mapping system for my life archive project.

I forgot to mention this, but recently (yesterday) our goat `Betty` had a baby. Luckily it wasn't a stillbirth this time. This baby is really cute (in my and everyone else in my households opinion) and it has been fun taking care of it. Today, the baby goat got to bask in the sunlight for the first time. It hasn't even been 48 hours yet, and the goat is already running and hopping around. I have gotten 2 videos of the goat today. I am really considering getting back on Vimeo to upload my videos for viewing somewhere.

Today was a good day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 8th 2021

I had a good day with development today. However, I have been really unfocused today, and my work has been all over the place. I broke my rule on caffeine addiction, as I desperately needed the boost today. I don't think it actually helped though, so I wasted a 3 month streak.

My laptops keyboard is getting worse again, and so is the battery. The `A` `S` `F` `FN` and `LEFT CTRL` keys have been having problems on and off today. I hope it is like March 2021, and these issues go away on their own. About my laptop battery, the capacity dropped another 6% today, and is now down to 77.8%, a 22.2% drop since July 9th 2020.

I started a new tradition of doing more forks of other projects today. It is helpful for note organization. I found that a few of my favorite old Minecraft mods are on GitHub, such as MrCrayfish's furniture mod, railcraft, and one I never installed but have seen lots of footage of: `tooMuchTNT`

I continued work on my degoogle campaign today, and made progress in modernizing some of the projects. I plan to do more updates tomorrow if I have time. I can't believe its Friday again already.

I am really working on going to bed earlier again today, so I quit earlier than last night. Tomorrow might be a difficult day if the addiction is bad again.

Today was a good day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 9th 2021

I had a pretty difficult day today. Last night, I got less than 7 hours of sleep, and I experimented with caffeine further, and have decided to permanently ban myself from drinking it due to bad reactions I had from it today and yesterday. This affected my day in both a positive and negative way. Mentally, it was negative, positively, I had more time, and got more done, but still didn't get everything I wanted to do done.

I got the normal work done with Git-image work, but had it even worse with my game addiction problem today. I have a last day of difficulty tomorrow before I have 5 days of normalcy.

I did further work on my degoogle-your-life campaign today, and left off 1 task (archiving 9 issues) due to little time.

On my own, I have been very social today.

Today was an OK day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 10th 2021

I had a very good day, but also a poor day in some ways. I was doing much better mentally today. I got a decent amount of work done today and made progress in current projects.

For image work, I came across more old, unuploaded content. I started uploading more of it today.

Today with personal time, pCloud completely failed, and no longer works for me on either my phone ot laptop. I tried Degoo, but it was still broken. I can't keep doing this, so I am just going to use an SD card instead.

I was going pretty fast with my work today, and received 2 GitHub `500 Internal Server error` errors in less than 4 hours, a new record.

I am still a little peeved about GitHub disabling the dark/light mode switch on my profile page, I have to keep an additional tab open again just to get the brightness right.

I hope to make improvements to work through the next 5 days. I don't know what else to say.

Today was an OK day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 11th 2021

I had a good day with work today, but my night didn't go well mentally.

Today, I learned another cool feature of markdown, which is the creation of checkboxes. The syntax for an unchecked box is `- [ ]` and the syntax for a checked box is `- [x]` pretty simple. The only con I can see to its usage is that you can't check/uncheck the box without modifying the file each time. I like this feature, and I am trying to find a way to use it.

I worked heavily on 2 political projects today, as I had ideas for them, and they needed to be finalized (for now)

Today was day 2 of the SD card transfer on my phone, I noticed that 2 files got corrupted in the process, but enough of their data remained that I feel like they can be repaired (unlike Windows, where the corrupted file will be 0 bytes in size every time) I feel like only a few lines were corrupted, and I feel these files are repairable. It isn't too much of a big deal, unless it gets worse, and the corruption ratio gets far worse than 2/1503 files being corrupted.
 
I got a lot of documentation work done today, along with the usual work. I hope to get something unique done tomorrow, but right now, I have to go to bed a little earlier (day 1, attempt ?/2021 of attempting to fix my schedule again) 

Today was an OK day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 12th 2021

I had a good day with work today, and my night went better.

Microsoft brought back the GitHub light/dark mode toggle on the profile page today, I like it better like this. Hopefully it will stay this time.

I did my usual work processes today, but missed out on cleaning. I am working on going to bed earlier and earlier to fix my schedule. An appointment today really threw me off.

Today during SD card transition, there were no corrupted imahge files today, although I have been noticing that each time I sort the batches that I copy off the SD card, the tracker-store daemon activates, and starts going between CPU cores and using 12.5*8% CPU for up to an hour.

I did brief Political work today, there was something in wanted to add.

My laptop had severe issues with the S and A keys today, these later were corrected. I don't know what keeps happening.

I fixed the tracker-miner-fs issue on my backup laptop today, and made a few edits from this device. I have enjoyed the 20000+ media files I can rediscover.

I am working on backing up my Reddit collection. I am even considering putting some of the content online early, although no NSFW content (gore, sexuality, etc.) will not be uploaded yet, along with 25+ megabyte files. It is a very sensitive set.

My laptop has been having battery issues related to the tracker daemon coming on over and over.

Last night, I accidenally forkbombed myself with a file I made in the past. It was a RAR file that contained 3 images (less than 4 MB) but it resulted in a 23+ gigabyte file, and bringiny my system down to 0 bytes of memory, so low that the file system menu refused to open, and I couldn't save empty files (testing)

Today was an OK day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 13th 2021

Today was a very difficult day for development. Personally, I had sleeping issues, and I have been feeling underappreciated lately. I don't know where to socialize with people who are as into tech as me.

My keyboard completely broke today, the A key came off completely, and putting it back on after cleaning still resulted in issues. I would just use the button under the keycap, but I don't want to break that as well. I had some issues, but I transitioned to my backup laptop, the whole reason we got it was for issues like this. I have had distractions with the backup laptop lately, I have been doing a massive sort of files from Reddit. There are still some steps to get it to be a functional backup and half clone.

On GitHub today, I had a poor day with development and wasn't able to get part 2 work (`GitHub-daily` `Git-images`) done. I also couldn't get very much done today. I got part 1 work done, and worked on a new article repo (year of the Linux desktop) which I put a lot of work into. I didn't get very much done today.

Today was not a very good day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 14th 2021

Today was a very difficult day for development. I spent nearly 3 hours sorting pictures this morning, before starting my gaming sessions. I was so out of it last night that I fell asleep on the armchair last night, unable to make it to my own bed. I don't know how I ended up there.

I have been doing documentation work all over the place today after finishing sorting Reddit media and completing my short gaming sessions. I have limited music to listen to on my backup laptop right now, which makes work difficult.

Today was my first full day of backup laptop usage. I had to call tech support twice today, first for the issues with the other laptops keyboard (which is expected to get repaired on April 17th 2021) and also to try and find out about the error `System program problem detected` meanwhile during the call, I found that after making a backup of some data on my old laptop, my new laptop was refusing to mount the hard drive. The issue got worse when I found out both Linux laptops couldn't mount the laptop. It became a panic, and I am sorry to tech support if my upsetness was shown on the call. This has been a deathly fear for 5 years, and it finally occurred today. It is going to add constant stress until I get a backup of the backup (3.6 TB) luckily it isn't the drive that is the problem, as the drive is fully functional on a Windows device with a USB port without a dongle.

This issue caused lots of suffering today, lately, everything wrong has been happening, and I have dealt with more and more severe errors every day. It has been a nightmare today. I tried to distract myself, but it kept coming up. Later on, I tried to copy data from an SD card on the dongle on the first laptop again, and the moment I tried to copy a second stream of files at once, it got severely corrupted and made the file system read-only for some reason. This caused further panic, and I can now further rule the dongle may be a problem after tech support. I honestly need more people from my own world of thinking (people who are highly technically savvy) to help me out, but I can't find anyone.

I did some note work on GitHub today, along with some of the usual tasks, except I skipped the `GitHub-daily` (Git-image) work again today.

I have been working on fixing my schedule lately, and it has gotten too out of control, so I had to give up on it again today. I will try to get it under control again starting Sunday.

Today was not a very good day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 15th 2021

I am back today. I don't feel fully comfortable yet with my misconfigured setup, but I am getting used to it. I am really getting fond of this laptop.

I was really tired while working last night and have been failing on doing my most basic daily tasks. Today, I finally got caught up on the work I have been unable to get to lately. I also grabbed the opportunity and made the `Year-of-the-Linux-desktop` organization, which will have heavily Linux-related projects.

I still was a bit slow today, and I am still stressed about hard drive related issues. I have not been willing to test the 2nd dongle to see if it was a dongle-related issue causing hard drive and other external device errors.

I had a much better day with development today, and I managed to finalize some work. I hope to try and do project work tomorrow, although I have some games I like to play on Fridays, and I have guests coming over, so I might be booked.

Today was a good day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 16th 2021

I had a good day with development today, although my world is still being turned upside down by technical problems. My new laptop had its first system crash today, along with other issues.

I did the usual tasks today, but really felt like I wasn't doing enough, I started a few projects today, and hope to get back into my degoogle campaign.

I have gained interaction in the inform programming language and its usage todauy.

I had guests over today, family members.

I have recently learned of Google FLoC and I want to write about it soon.


Today was a good day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 17th 2021

I had a good day with development today, although my world is still being turned upside down by technical problems. My new laptop didn't crash today, and I found out how much more helpful Linux is comparted to Windows on reporting errors. Linux gives you all the details, and additionally, I learned today that I can make the error redirect to a bug triage page about the issue. I would never have learnt this if I kept error reporting on by default.

I did the usual tasks today, and felt like I did a little bit more today. I was working quite a lot with SNU programming tools today, adding necessary libraries for languages, starting with a 2nd language (Inform) it is fun for me to do.

I started experimenting with the calming blue theme idea I had, but went in a different direction for now compared to the advanced features I had planned. You can see the theme in use on my later screenshots today. Yes, it does look tacky, this is just the first prototype. I am nicknaming it "Egypt B1" (B as in Blue, not Beta) because the zigzag pattern reminds me of Egypt for some reason. Currently, I am just experimenting with it psychologically, and seeing what colors work and what don't. I made sure not to include any yellow or red, as it is commonly associated with negativity (yellow as in error messages, such as ⚠️ red as in error messages 🛑 as well, along with low battery, and other stressors) I have noted that the menu (not shown in screenshots as of yet) is hard on my eyes, but for whatever reason is emitting a calming feeling to me. I don't know if it is my own bias or not. I will need to do further testing on this. Despite its tackiness, it still serves its purpose for me.

Based on review from one other person, an anonymous family member with vision problems, it has been noted that the green search bar is a problem as well. I never noted it to be a problem. Luckily, this is just a prototype, these issues can be worked out.

I stayed up really late again tonight, but got all my work done.

Today was a good day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 18th 2021

I had a good day with development today, but I didn't get everything I wanted to get done, completed.

I got a lot of experience with Linux today, learning several new terminal commands and tricks, and I installed Samba today. Unfortunately, due to 2 system crashes in less than 2 hours, where the mouse completely stopped working, I had to uninstall Samba. I received another crash about 6 and a half hours later which I assume was from VLC, as I only had VLC open for less than an hour, and it was open earlier as well. Unfortunately, this laptop is showing a lot of software problems of a past laptop. Luckily, my current main laptop is getting repaired tomorrow.

I spent a lot of time gaming today, but I also got some work done. Some projects I didn't get to today that I wanted to get to was improved support for the Inform IDE, starting the FLoC article, and optimizing my marine biology project, along with some project Wiki work. I will do this casually across the following week.

I was in a rush to finish tonight, but I took a break, as one of our goats gave birth to 2 children tonight, and I wanted to go out and see it and pause from my work.

I got a decent amount done today, and I finished doing documentation work later on.

I stayed up really late again tonight, but got nearly all my work done.

Today was a good day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 19th 2021

I had a difficult day today. I got my laptop keyboard fixed on my main laptop, but due to my laptop not having a CMOS battery, the motherboard lost power and reverted the BIOs to default settings. I can't get it to boot past BusyBox. I was on call with Dell for over an hour today, and got to their 2 solutions of wiping the operating system and losing everything (which I won't do) or reach out to a 3rd party. I need help, and I am willing to pay cash for help (current offer is $40, the offer may go up)

When/if I start manufacturing computers and laptops in the future, I am going to make sure a CMOS battery or a fully functional tested alternate to it is present in every device I manufacture. 

I have joined ProtonTechnologies (the creators of ProtonMail, ProtonVPN, and more) today in supporting the people of Myanmar. I have now made Myanmar/Burmese a top priority when translating an article. I will keep this decision, even if the language were to go extinct. This is a permanent decision.

I finally got to one task I wanted to do yesterday today, which was working on the anti-Google FLoC article. I created a good article in good time, and started translating it. I did heavy amounts of translation to it. I added 2 new organization projects today, with only 1 being able to be made by me this month, due to a set limit of 10 per month.

I stayed up really late again tonight, but got nearly all my work done. I worked on trying to go to bed earlier

Today was a good day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 20th 2021

I was really tired last night and I didn't get to uploading my status post on time. I did that today.

Today was a pretty good day for development. I started 3 new research and documentation projects (although the GitHub Linguist hasn't picked up on the first 2 for some reason) but mainly today, I did translation work. I manually translated my Degoogle article into 50+ different languages, and set up importance of translations (which translations to do first)

Recently, I have noted that I have been working like I am not going to die tomorrow. So I have been trying to get important documentation tasks done as well to explain my projects better in a way that only I know.

I have had no luck in repairing the laptop, and my computer just nearly crashed at 11:21 pm, I saved it this time. VLC may be the problem, or it might be something else. I can't figure out what.

I got a follow-up with Mozilla today on my question/concern. Apparently, the theme does work, so I am going to have to add instructions on getting it to work.

I went to bed a little earlier tonight, and I was able to wrapup before midnight tonight. I could have done more today, but I needed the sleep.

I did work in various projects today. I did get a lot done, no matter what I falsely tell myself. It has been hard working without music lately though, as VLC crashes my laptop within 2 hours. 

Today was a great day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 21st 2021

I had another good day with development today, but I started out with a feeling of brain tension. I finally got through my small to-do list, and now I have gathered new assignments for myself.

I created 2 new organizations today. I was going to make just 1, and keep it at 10/month, but I decided to go for an extra one, as I felt it was needed, and now I have a full 7x7 square of organizations.

I worked heavily on SNU today, but mostly did heavy work on other documentation.

I am going to bed earlier, so I am going to quit here. I have an appointment in the morning.

Today was a great day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 22nd 2021

I had another good day with development today, this week has been pretty good. My schedule is starting to get out of control again though.

I discovered my old camera project today (CamCam+/CamCam++) and made extensive edits to its documentation. It hasn't been updated in over 5 months, and I got it up to date. I also spent over an hour doing translations, so it would rank higher in my commits and be another project type that is capable of being read in various languages all around the world.

I did work on various projects today. I started earlier on, as I got out of a meeting, and still had my laptop running on battery (not plugged in) it was still discharging, so I decided to use it and get some work done before starting my gaming sessions.

I made edits across various projects today, but was careful not to go to 25 different repositories today. I have a huge important personal event tomorrow at 1:00 pm that will last 1-2 hours, so I will have less time tomorrow, although I have been doing very well for the past 4 days, so that should make up for it. It might even cut off my Friday Egg Inc addiction, that and the excessive charge time of my phone.

I had to make an edit to my political repository today, as I re-read the sponsor rules on GitHub and had to check to make sure sponsorships weren't active on the project. In my head, I thought I laid out the rules pretty well that I didn't feel comfortable with that there, but I am glad I checked, as I was breaking ToS (Terms of Service) by having the button active. I have been avoiding committing to this repository, as it is not what I want to be known for, but I had to make 3 commits there to eradicate the sponsor button completely.

I added a very privacy-invasive project to GitHub today, but didn't develop it to a working point. I needed to get the idea out, and force public awareness that if you are to blur an image, you need to make it impossible to reverse. It is already pretty late to be doing this, but it has to be done, so that future info can be better protected. A good way to censor is to put opaque shapes over sensitive data, or pixelate it with very large DPI pixels.

Today was a great day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 23rd 2021

I had another very good day with development today. Again, this week has been a great week for development. I have been pushing really hard on my top 25 projects this month. My normal Friday addiction was cut short due to an important appointment, which I hope to try and do more often.

I had someone follow me today who claimed to be an EFF Member. I was pretty happy about it, but data from the next day shows they were likely just trying to harvest tons of followers, because their following (number of people they were following) dropped by over 600, and the EFF Member part was removed. They have since unfollowed me and haven't responded back.

I plan on working on my SVG video project, but the concept had to be partially rewritten overnight to take advantage of many features of SVG, and also to further difference between SVG Animation (it is not a fair comparison, it would be like comparing GIF to MP4. They are different formats that serve different purposes.

My computer remained relatively error free today, I still want to go back to my main laptop and keep this one as emergency backup. I didn't have time and also forgot to write a status file today, so I did so the next day.

Today was a great day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 24th 2021

I had another very good day with development today. It wasn't as good as the past 5 days though. My laptop had a significant number of errors today regarding the cursor freezing up, and I tried several things to fix it. So far, the working solution was to close the tab that had ~90 SVG video thumbnails (none of them were playing, but the laptop hasn't crashed in over 3 hours, so I assume this fixed it for now, I exhausted every other solution I could find)

I got back on top of status file and documentation work today, and also improved the SVG video format project. I did some work in other projects as well, but I mostly focused on the SVG_Video project today.

I quit slightly earlier tonight. I intend to work on my schedule again tonight.

Today was a great day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 25th 2021

I had another very good day with development today. This week has been really good.

I didn't work very much on the SVG video project today, but I worked in my usual projects, along with 1 new project regarding technical support of a digital cursor acting up. It will be a reference to myself and others. In this project, I made a joke that I ended up regretting when I went to share the project with my family, which was a criticism of the Windows 10 update process. I still stand by it, as it is still a major problem, and is one of the main reasons why I switched to Linux, and is my most easy to justify criticism of Windows 10 and switching to Linux (having control of your system is a good thing, I and many, many others have lost time and progress due to forced updates from Microsoft) I have partially corrected the joke, as it was meant to be serious documentation, but it will take time to purge it from the main translation README file at root, and all the other translations.

Today is my 11 month anniversary of GitHub usage. In 1 month, I will be a GitHub user for 1 year.

Today was a great day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 26th 2021

I had a difficult day today. I got some work done, but I didn't do very well. There were a lot of factors, but the main one was likely getting less than 6 hours of sleep, as the past 3 times I went with less than 8 hours of sleep, I haven't done well.

Today, I finalized and began publishing a new concept that I have been developing and deciding on for 2 months, which begins the Skeuwie (or Skeuowie) desktop environment, the UNIX environment with a sole focus on skeuomorphism (the environment is heavily inspired by KDE, but I am going in a different approach design and functionality wise)

I discovered GitHub skyline today, and I find it pretty cool. I would experiment with it and have my 2020 skyline as a trophy, but unfortunately, my familys 3D printer has been out of service for quite some time (at least 6 months) and it might be a waste of filament, but I feel that it would be a good trophy as a first year of GitHub.

My laptops battery has had unusually poor performance today. I hope I can resolve the issue soon.

I did GitHub work during my Khan Academy session and continued to listen, as I have already gone in-depth with this current subject in 10th grade biology class.

Today was an OK day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 27th 2021

I got a lot done again today, but still slightly less than what I wanted to do.

I took a couple hours out of my time to update my cursor tech support article and update and add many translations.

India has been having a nightmare lately. They recently reached 500 million COVID-19 cases, and they are experiencing a spike of cases over 10x of that of their May 2020 outbreak. I feel horrible about this, and I am doing what I can to make the lives of the people in India easier. There are so many beautiful cultures in India that I don't want to die out. Similar to the Myanmar situation, I have made India a foreign aid program member, and I am now prioritizing various Indian language translations for projects (such as Hindi, Bengali, Punjabi, Tamil, Telugu, Gujarati, and many more)

I have really come to dislike Firefox 89. From the beta, the Proton tab system is heavily flawed, and Firefox is losing more and more user control. Even Firefox 88 has some problems, due to some things getting moved around and renaming (like the close tab button being moved up 2 spaces on the dropdown menu. or `customize` being translated to `customize toolbar` which confused me at first, as I wasn't trying to modify the toolbar) I have 3 backups lined up for if it gets worse (Tor with ProtonVPN, Konquerer, or Waterfox)

I worked on various projects today. I am dreading next Tuesday, due to my Idle Miner Tycoon session being a possible setback and addiction on that day.

I did some development on WacOS today, it is my top programming project. I didn't do much to it today, mostly documentation changes.

I have continued to procrastinate on the development of the SVG video and Skeuowie projects, along with LinkLax.

I have come up with an idea, where projects with generic titles will become templates, and will be used to create a life archive extra. Also, a planned commit description change that I hope to keep and not forget is to say `for this directory set` when referring to a root file clone in a folder. It is late at night, I don't know how to simplify this. I hope to continue this change and not forget.

Today was a very good day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 28th 2021

I had a good day today. I had a very long set of gaming sessions this morning before starting work, and I enjoyed my work a bit more today. I worked further on the SVG video project, and also some other projects, such as SNU Translate, and LinkLax.

I torrented a couple files today, archive.org appears to be a safe place to torrent from, as the only peers are web seeds, and not random Internet users. I am still not going to do anything "illegal" though.

I had continued issues with my laptops battery today, but they weren't as severe. I would like to promote VLC for Android for those who can't degoogle and are stuck with a Samsung ROM, as VLC for Android is really incredible. The desktop version has a lot of functionality, but it nearly at par on the mobile version, there are some pros that the PC version doesn't have, like the scanning and parsing/guessing of your songs with high accuracy (without having to reference the Internet) it can tell the genre of all your music somehow. It has really good control, and I have genuinely been enjoying the experience lately. Please note that I am not affiliated with VideoLan, but I wouldn't mind an affiliation.

So lately, on GitHub, I have been focusing on 4 things:

1. Making more quality commits

2. Working on the listing of my top 25 projects of the month

* 1. Pushing /politics as far to the bottom as possible, since I don't want to be known solely for it

* 2. Getting projects on the list with more and more commits, so that as many projects as possible have 100 or more commits without a loss in quality

Today was a good day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 28th 2021

I had a very good day today. I completed the rest of my non-consecutive monthly goals (other than image projects) and I got a lot done today in various projects.

One of my big goals was recently completed, which was increased software library support in SNU Translate. I made a lot of progress with this today.

GitHub had a weird error today and signed me out. It was the only error that occurred today.

I have been waiting for May for the past 9 days. I really want to see my month in review, and get closer to my 1 year anniversary, and have a clean slate of progress.

I got my work done much earlier today, and stopped working by 9:00 pm. I have an early doctors appointment tomorrow morning, and I need to get my schedule under control again. I have really been failing on this lately.

I have done my research, and the problem I have with Firefox performance has not been fixed at all. CPU usage may be down with Firefox Quantum, but I have come across various posts from people with my exact issue (Firefox using more and more RAM overtime) from my research, it has been a problem since at least Firefox 3.6.x and continued, even into the Quantum era. CPU usage has been generally bad today when playing videos, jumping 40% during my Khan Academy work. I hate doing this, but I am going to have to switch to another non-Chromium browser. It sucks, as I have been using Firefox for most of my life when browsing the Internet (although I did use Internet Explorer 6 in my early years of Elementary School, and I was on and off Google Chrome from 2009 to 2018. I just don't like what Mozilla is doing, and I need a better browser. I am still trying to decide between Tor, Waterfox, and Konquerer.

I did another torrent today, my 2nd largest one since the entirety of the English Wikipedia, and other Wikimedia subprojects in English (surprisingly, all of that is only 190 gigabytes) todays torrent was over 29 gigabytes. I have now noted that archive.org isn't completely safe, as it is still peering to other people on large files, but I decided to take the risk today.

I have learned why I have been having difficulty writing scripts lately (subject "scripts" for "code" if the terminology is incorrect, I really just hate the word "code" and "coding" and prefer "program" "programming" "software engineering" "script" and "scripting" instead) which is that the reason I stop myself is because I start writing boilerplate code, which is pretty pointless to have everything in. This is why I stopped myself on work on some projects (such as SVG Video) until I find new methods of programming.

Today was a good day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: April 30th 2021

I had a very good day with development today. Today is the last day of April 2021, and I have a clean slate of progress tomorrow.

This morning, I had a doctors appointment, it didn't go too well (in my opinion yesterday, but today, I realize it was important) and I got another vaccine (not a COVID-19 one, as I already had both of them earlier this year) I am now fully up-to-date on my vaccinations (including COVID-19 vaccination)

I did less GitHub work today. I also didn't have the time to write a daily status post, as I got too tired. I did more torrenting today, but one got stuck, so I just went and downloaded from archive.org, as the site appears to be functional now (it was failing 4 megabytes in on downloads the last time I tried to download directly) I did a lot of work today, and was in a really good mood again today.

My laptop hasn't been doing too well. I recently fully uninstalled Samba, but it is still having severe battery drain issues, along with random freeze-ups (as of May 1st 2021, I have noted that waiting 1 minute or pressing the shift key will resolve the freeze-up issue)

Today was a good day for development.

***

**Comment by seanpm2001** [comment]
***

# End of month

### Status files are no longer being added here. See #9 for more daily status posts (For May 2021)

***
